docs/sphinx/source/api.rst

```diff
@@ -148,6 +148,7 @@ Transposition models
    :toctree: generated/
 
    irradiance.total_irrad
+   irradiance.get_sky_diffuse
    irradiance.isotropic
    irradiance.perez
    irradiance.haydavies
```

Comment:
Perhaps naming consistency is better here. Only one function has `get_` prefix.

Reply:
In this module, yes, but we use ``get_`` for a few other "wrapper" functions/methods in ``solarposition.py``, ``location.py``, and ``pvsystem.py``. I am torn on this.

Reply:
I'm not in favor of 'get_isotropic', an exception to my preference for verb prefixes.  isotropic, perez, haydavies are named models.  If we want to extend the function name, I'd use a category prefix, i.e., sky_diffuse_isotropic, sky_diffuse_perez, etc., and have get_sky_diffuse wrap all of them with a kwarg.